src/foundry/foundry.js/roll.d.ts

```diff
@@ -69,7 +65,7 @@ declare class Roll<D extends object = {}> {
    * Cache the numeric total generated through evaluation of the Roll.
    * @defaultValue `undefined`
    */
-  protected _total: number | undefined;
+  private _total: number | undefined;
```

Comment:
We agreed on the middle ground of making things that are marked `@private` in Foundry, `protected` in our sources. So there is no need to change the access modifiers.